### DISCUSSION TITLE
Replace .jpg images with different, lossless format?

### DISCUSSION THREAD

**Discussion by jdietz224** [RayTracing/raytracing.github.io#1031] [General]
In many different parts of the books one of the points of different renders is to see how the amount of noise in the image changes with different changes in the code.

I've always felt like my images have looked to my eye like they have more noise in them than the images in the book and that's made me wonder if I'm doing something wrong.

Then I noticed that the images in the book are .jpg images. Is it possible that the .jpg compression is reducing the amount of high-frequency noise in the image? If so, it might be more clear to include some lossless image format so that it's easier to compare my results with the book's.

**Comment by trevordblack** [comment]
Hmm.

We've thought about this before.
We decided between png and jpeg for the renders, but ultimately settled on jpeg because the time it takes for the website to render is important to us.

Changing to a lossless format would represent a significant footprint in terms of page load time and memory.

Something to think about

**Reply by KnockerPulsar** [reply]
Perhaps having a folder of reference images tucked away somewhere for anyone needing to pixel peep would be enough? I've also had this issue.

**Comment by hollasch** [comment]
_Almost_ all render images are (lossless) PNG format. We go with JPG where readers don't need to compare with their own results. Still, a few of our render images are still JPG. These will be converted to PNG during the v4.0.0-rel milestone (https://github.com/RayTracing/raytracing.github.io/milestone/19), where we will also be updating the renders with the latest changes.
